common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -201,11 +207,10 @@ def test_answer_incorrectly(self):
     @attr(shard=7)
     def test_submit_blank_answer(self):
         """
-        Scenario: I can submit a blank answer
```

Comment:
@alisan617 Please take a look at this.  I do not think we can submit blank answers any longer, but I want to make sure.  I update this test to match the current working state of submit.